scripts/ec2_bootstrap_al2.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Copyright 2024 Amazon.com, Inc. or its affiliates.
+#
+
+#############################################
+# EC2 Bootstrap Script for OSML Demo
+#
+# Installs all necessary dependencies on a fresh EC2 instance to deploy the OSML demo.
+# Known good configuration:
+#     - OS: Ubuntu 22.04 LTS (AMI ID: ami-08116b9957a259459)
+#     - Instance Type: t3.medium
+#     - Root Volume: 50 GiB gp2
+#
+# Usage:
+# Insert this script into EC2 User Data or run as root on an EC2 instance with internet connectivity.
+#############################################
+
+# ASCII Art Header
+cat << "EOF"
+ _______  _______  __   __  ___
+|       ||       ||  |_|  ||   |
+|   _   ||  _____||       ||   |
+|  | |  || |_____ |       ||   |
+|  |_|  ||_____  ||       ||   |___
+|       | _____| || ||_|| ||       |
+|_______||_______||_|   |_||_______|
+ _______  _______  _______  _______  _______  ______    _______  _______
+|  _    ||       ||       ||       ||       ||    _ |  |   _   ||       |
+| |_|   ||   _   ||   _   ||  _____||_     _||   | ||  |  |_|  ||    _  |
+|       ||  | |  ||  | |  || |_____   |   |  |   |_||_ |       ||   |_| |
+|  _   | |  |_|  ||  |_|  ||_____  |  |   |  |    __  ||       ||    ___|
+| |_|   ||       ||       | _____| |  |   |  |   |  | ||   _   ||   |
+|_______||_______||_______||_______|  |___|  |___|  |_||__| |__||___|
+EOF
+
+## Variable definition
+LOG_FILE="/var/log/user-data.log"
+USERNAME="ec2-user"
+
+## Enable logging for startup script
+exec > >(tee -a $LOG_FILE)
+exec 2>&1
+
+## Install the AWS CLI
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
+
+## Install dependencies (root level)
+yum install -y git-lfs python3-pip aws-cli npm docker
+
+## Start and enable Docker service
+systemctl start docker
+systemctl enable docker
+
+## Install CDK
+npm install aws-cdk
+
+# Clone primary AWS repo
+git clone \
+    https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws.git \
+    && cd guidance-for-overhead-imagery-inference-on-aws \
+    && git-lfs pull
+
+chmod -R 777 guidance-for-overhead-imagery-inference-on-aws
+
+# Clone submodules from HTTPS instead of SSH (since git ssh keys are not available yet)
+cd guidance-for-overhead-imagery-inference-on-aws \
+    && sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules \
+    && git submodule update --init --recursive
```

Comment:
Replace all `guidance-for-overhead-imagery-inference-on-aws` with `guidance-for-processing-overhead-imagery-on-aws`